atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/TargetedSweeper.java

```diff
@@ -114,6 +117,11 @@ public void initializeWithoutRunning(TransactionManager txManager) {
         initializeWithoutRunning(SpecialTimestampsSupplier.create(txManager),
                 txManager.getTimelockService(),
                 txManager.getKeyValueService(),
+                TransactionServices.createTransactionService(txManager.getKeyValueService(),
+                        CoordinationServices.createDefault(
+                                txManager.getKeyValueService(),
+                                txManager.getTimestampService(),
+                                false)),
```

Comment:
discussed offline: does not need to be async, since it will be tried when the requirements are ready